web/src/components/navbar/desktop/NavBarDesktopDropDown.tsx

```diff
@@ -0,0 +1,98 @@
+import { Icon } from "@/components/njwds/Icon";
+import analytics from "@/lib/utils/analytics";
+import { ClickAwayListener, Grow, MenuItem, MenuList, Paper, Popper } from "@mui/material";
+import { ReactElement } from "react";
+
+interface Props {
+  disabled?: boolean;
+  anchorRef: React.MutableRefObject<HTMLButtonElement | null>;
+  open: boolean;
+  textColor: "primary" | "base";
+  menuButtonTitle: string;
+  dropDownTitle: string;
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  handleClose: () => void;
+  icon?: ReactElement;
+  subMenuElement: ReactElement[];
+}
+
+export const NavBarDesktopDropDown = (props: Props): ReactElement => {
+  const toggleDropdown = (): void => {
+    if (!open) {
+      analytics.event.account_name.click.expand_account_menu();
```

Comment:
pebble - are we testing analytics firing? Should we move it to the relevant test file?

Reply:
ya